proxy/http/HttpConfig.cc

```diff
@@ -938,6 +965,21 @@ HttpConfig::startup()
                         c.oride.server_session_sharing_match);
   http_config_enum_read("proxy.config.http.server_session_sharing.pool", SessionSharingPoolStrings, c.server_session_sharing_pool);
 
+  RecRegisterConfigUpdateCb("proxy.config.http.insert_forwarded", &http_insert_forwarded_cb, &c);
+  {
```

Comment:
Rather than duplicating this code, could you just invoke the callback?

Reply:
I guess I could, is that what you'd prefer?  The advantage isn't clear.

Reply:
Code duplication should be avoided if it can be done without strong contortions.

Reply:
It can't be done because of the call to http_config_cb at the end of http_insert_forwarded_cb.

Reply:
OK